git/git_branches_intro.md

```diff
@@ -1,22 +1,28 @@
 # Git Branches Introduction
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+This lesson is focused on one of the key features of Git: branches. Branches allow developers to create a fresh copy of their repositories and experiment. Branches can be useful for testing new ideas, compartmentalizing projects, and allowing multiple developers to contribute to the same project simultaneously.
```

Comment:
> Branches allow developers to create a fresh copy of their repositories

I agree this is a good way to describe branches, but I think it might cause some confusion with what a fork is. I wonder if theres another way to describe what a branch is?